src/iris/bin/sender.py

```diff
@@ -1052,13 +1056,14 @@ def fetch_and_send_message(send_queue, vendor_manager):
         message = send_queue.get(True, 4)
     except queue.Empty:
         return
+    blame_str = 'app:%s  target:%s' % (message.get('application', '?'), message.get('target', '?'))
 
     metrics.incr('send_queue_gets_cnt')
 
     retry_count = message.get('retry_count')
     is_retry = retry_count is not None
     if is_retry and retry_count >= MAX_MESSAGE_RETRIES:
-        logger.warning('Maximum retry count for %s breached')
+        logger.warning('Maximum retry count for %s breached', blame_str)
```

Comment:
Why use a separate variable instead of something like:

    logger.warning('Maximum retry count for app: %s target:%s breached', message.get('application', '?'), message.get('target', '?'))